mibidata/mibi_image.py

```diff
@@ -229,10 +236,20 @@ def aperture(self):
     @aperture.setter
     def aperture(self, value):
         if value is not None and value not in APERTURE_MAP.values():
-            raise ValueError(
-                'Invalid aperture code \'{}\', must use values'
-                'from the following map: {}'.format(value, APERTURE_MAP)
-            )
+            if value in _DEPRECATED_APERTURE_MAP:
```

Comment:
Great. In the future we can look to simplify `libted` pipeline app by removing this same logic from the scripts there.